index.d.ts

```diff
@@ -1532,6 +1532,7 @@ declare namespace Eris {
     editSelfSettings(data: UserSettings): Promise<UserSettings>;
     editStatus(status?: Status, game?: ActivityPartial<BotActivityType>): void;
     editUserNote(userID: string, note: string): Promise<void>;
+    editVanity(guildID: string, code: string): Promise<{ code?: string; uses?: number }>;
```

Comment:
Can the return type have its own interface?

Reply:
Also are both return props not guaranteed?

Reply:
Not sure, there are no official api docs for this. But I saw it was partial on the getVanity method.